src/masonite/tests/TestCase.py

```diff
@@ -138,12 +147,12 @@ def fetch(self, route, data=None, method=None):
         for name, value in self._test_headers.items():
             request.header(name, value)
 
-        route = self.application.make("router").find(route, method)
+        route = self.application.make("router").find(path, method)
         if route:
             return self.application.make("tests.response").build(
                 self.application, request, response, route
             )
-        raise Exception(f"No route found for {route}")
+        raise Exception(f"No route found for url {path}")
```

Comment:
Instead of printing `No route found for None` we get `No route found for /not-existing`.  `route` which was None in L150 was shadowing `route` arg provided to the function